src/app/client/src/app/modules/core/services/copy-content/copy-content.service.ts

```diff
@@ -174,7 +191,10 @@ export class CopyContentService {
       }
     } else if (contentData.mimeType === 'application/vnd.ekstep.ecml-archive') {
       url = `/workspace/content/edit/content/${copiedIdentifier}/draft/${contentData.framework}/Draft`;
-    } else {
+    } else if (_.get(contentData,'mimeType') === 'application/vnd.sunbird.questionset') {
```

Comment:
let me try this

Reply:
@vinukumar-vs 
I did the analysis,  by solving one function problem. if we are importing the workspace module into the core module. it will be imported into two different files. which will be a bumping of the bundle size. it will solve only the if-else method problem. we are not using any other method from workspace modules in core modules. 
and also some code-level changes are required to call this workspace .service.ts
Required parameter **state**: unable to find this metadata on click of copy content/questionset
need to add two additional metadata to call this service [identifier,status] 

CC: @rajeevsathish @rajeshkumaravel